testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDeleteCompatibility.java

```diff
@@ -267,7 +267,7 @@ public void testDeletionVectors(String mode)
                     .containsOnly(row(2, -1), row(30, -1));
         }
         finally {
-            dropDeltaTableWithRetry("default." + tableName);
+            System.out.println(tableName);
```

Comment:
pls drop debug content

Reply:
Dropping is intentionally disabled to debug the contents.